parlai/crowdsourcing/tasks/turn_annotations_static/turn_annotations_blueprint.py

```diff
@@ -184,18 +189,31 @@ def get_frontend_args(self) -> Dict[str, Any]:
         for use by the task's frontend.
         """
 
+        # Load task description from file
+        MISSING_TASK_DESCRIPTION_TEXT = (
```

Comment:
hmm what is the advantage of defining `MISSING_TASK_DESCRIPTION_TEXT` and then defining `task_description` as equal to it?